scss/civicrm/contact/_tabs.scss

```diff
@@ -9,11 +9,18 @@
     display: flex;
     padding: 0;
 
-    .ui-tabs-nav {
+    &:not(.ui-tabs) {
```

Comment:
This specific code is used to hide the panel temporarily until javascript does not generate `.ui-tabs` .
Once the javascript is kicked in. this style stops getting applied and tab works as it is.